README.md

```diff
@@ -21,6 +21,49 @@ framework
 * [ReportPortalPortal](https://github.com/znsio/ReportPortalIntegration/blob/main/README.md)
 * [ApplitoolsIntegration](https://github.com/znsio/ApplitoolsIntegration/blob/main/README.md)
 
+## Pre-requisites for executing App Tests
+
+**Steps To Install Appium npm and Node.js**
+
+1. Before installing the Appium we need to install the Node.js and npm
```

Comment:
We don't need steps to install node. Just give a relevant reference link

Reply:
Removed